modules/dev-tools/config/ocular.config.js

```diff
@@ -5,7 +5,11 @@ const IS_MONOREPO = fs.existsSync(resolve('./modules'));
 
 const DEFAULT_CONFIG = {
   babel: {
-    plugins: ['version-inline']
+    configPath: getValidPath([
+      resolve('./babel.config.js'),
+      resolve('./babelrc'),
```

Comment:
I would drop support for this.

Reply:
We still have a few repos that use `.babelrc`. It would be confusing if it's silently ignored.